airflow/www/extensions/init_security.py

```diff
@@ -31,12 +31,11 @@ def init_xframe_protection(app):
     See also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options
     """
     x_frame_enabled = conf.getboolean('webserver', 'X_FRAME_ENABLED', fallback=True)
-    if not x_frame_enabled:
+    if x_frame_enabled:
         return
```

Comment:
Hmmmm, does "correcting" this now end up breaking things I wonder?

i.e. do we follow the what our docs say, or do we update the docs to match what the implementation currently does?

Reply:
Oh this only landed in 2.2.5, right.

Reply:
Pre 2.2.5 it was actually worse because the flag work opposite of its description.

Reply:
Obligatory XKCD https://xkcd.com/1172/

Reply:
Very, very much this.

Reply:
Anyway -- I think this warrants a news fragment please.

Reply:
Absolutely :). The pleasure is mine. But I am not entirely sure what to write :).

"Finally it works as expected" :D ?

Reply:
I made an attempt to describe it - but it's a bit wordy :).